transform/transformer.go

```diff
@@ -1,6 +1,10 @@
 // Transforms static content form the 'templates' folder
 // in the root of the repository to Go code in order to
 // distribute it as a part of the module.
+
+//Noted: run this script in the transform dir after u changed the templates content
```

Comment:
I would omit usage of casual English in documentation. Let's remove `Noted:` and rephrase the wording in order to sound more like a documentation.

Reply:
I got u.